h/api/storage.py

```diff
@@ -33,15 +34,15 @@ def annotation_from_dict(data):
     return models.elastic.Annotation(data)
 
 
-def fetch_annotation(request, id, _postgres=None):
+def fetch_annotation(request, annotation_id, _postgres=None):
```

Comment:
I'm good with this. My only plea is that we disable linter things that we don't follow, otherwise actually useful linter warnings get buried. We've a lot of rule disabling to do. Hoping to find some time to look into it at some point